_posts/2022-06-03-gaps-in-bitcoin-dev-funding.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "The gaps in bitcoin open-source funding"
+date:   2022-06-03 09:57:39 -0500
+categories: bitcoin funding grants
+---
+
+A few people reached out about my post on my [Guide for Grant Seekers](/bitcoin/funding/grants/grants-bitcoin-open-source/) with some critical feedback about promoting work for free culture. To clarify, this guide was not meant to be an ideal vision for the future, it was meant to reflect the present and help grant seekers achieve their goals.
```

Comment:
```suggestion
A few people reached out about my post on my [Guide for Grant Seekers](/bitcoin/funding/grants/grants-bitcoin-open-source/) with some critical feedback about promoting work-for-free culture. To clarify, this guide was not meant to be an ideal vision for the future; it was meant to reflect the present and help grant seekers achieve their goals.
```

Reply:
I would hyphenate "work for free" because it is the name of the culture we are referring to, it can stay as is in the other places it's used her.